src/sql-parser/src/parser.rs

```diff
@@ -1925,7 +1925,7 @@ impl<'a> Parser<'a> {
             Keyword::Kafka => {
                 self.expect_keyword(BROKER)?;
                 let broker = self.parse_literal_string()?;
```

Comment:
I propose despecialifying the `BROKER` clause and making it just another option!

```sql
> CREATE CONNECTION kafka_ssl
  FOR KAFKA
      BROKER 'kafka:9092',
      SSL KEY = SECRET ssl_key_kafka,
      -- ...
```

It also occurs to me that `BROKERS = [...]` should be an array-accepting option that is mutually exclusive with `BROKER`. Then we could get rid of the gross accidental behavior that you can comma separate multiple addresses in the `BROKER` clause.